### PR TITLE
chore: uki der certs in iso

### PR DESCRIPTION
**Pull request by frezbo**
Add the uki signing cert into iso.

Fixes: #8131

### CODE REVIEW

File: pkg/imager/iso/uefi.go [hunk 1]
```diff
@@ -167,6 +170,10 @@ func CreateUEFI(printf func(string, ...any), options UEFIOptions) error {
 		}
 	}
 
+	if _, err := cmd.Run("mcopy", "-i", efiBootImg, options.UKISigningCertDerPath, "::EFI/Linux/uki-signing-cert.der"); err != nil {
```
Comment (smira) [inline pkg/imager/iso/uefi.go:173]: is it some standard path? or just dumping something?

Comment (frezbo) [reply pkg/imager/iso/uefi.go:173]: no standard path, i initially put it under `keys/secureboot` but it seems OVMF doesn;t allow selecting from that path, only `EFI` and `loader` were allowed

Comment (frezbo) [reply pkg/imager/iso/uefi.go:173]: maybe i can do `loader/keys`

Comment (smira) [reply pkg/imager/iso/uefi.go]: loader is I guess for sd-boot loader

Comment (frezbo) [reply pkg/imager/iso/uefi.go]: yeh, but it's `loader/auto`

Comment (frezbo) [reply pkg/imager/iso/uefi.go]: does `EFI/Linux/keys/uki-signing-key.der` makes sense?

File: pkg/imager/iso/uefi.go [hunk 2]
```diff
@@ -149,6 +152,10 @@ func CreateUEFI(printf func(string, ...any), options UEFIOptions) error {
 		return err
 	}
 
+	if _, err := cmd.Run("mcopy", "-i", efiBootImg, options.UKISigningCertDerPath, "::loader/keys/uki-signing-cert.der"); err != nil {
```
Comment (smira) [inline pkg/imager/iso/uefi.go:155]: my only concern is - should we move it to `::EFI/` so that we don't create something which would confuse systemd-boot

**Review by smira — APPROVED** [review]

File: pkg/imager/iso/uefi.go (hunk 2 above)
Comment (frezbo) [reply pkg/imager/iso/uefi.go:155]: yeh, that works, so `::EFI/enrollment/uki-signing-key.der` or `::EFI/keys/uki-signing-key.der`?

Comment (smira) [reply pkg/imager/iso/uefi.go:155]: lgtm!

**Pull request merged**